cpp/src/core/library.cpp

```diff
@@ -153,4 +157,13 @@ legate::Library& get_library()
   return library;
 }
 
+bool get_prefer_eager_allocations()
```

Comment:
This is not intended to be a long term solution right?

Reply:
IMO, from a user perspective, users shouldn't to have to touch any parameters for streaming to work - it should just work. This is a demo so its fine for now.